src/pylibsshext/channel.pyx

```diff
@@ -15,14 +15,17 @@
 # License along with this library; if not, see file LICENSE.rst in this
 # repository.
 #
+import signal
 import time
 from io import BytesIO
 
 from cpython.bytes cimport PyBytes_AS_STRING
 from libc.string cimport memset
 
 from pylibsshext.errors cimport LibsshChannelException
+
```

Comment:
Did some linter want these formatting changes?

Reply:
It must have been isort.

Reply:
Removed.